### PR TITLE
[2.5] [Backport] Rancher logging upgrade to 3.14.0 and backports

### PR DESCRIPTION
**Pull request by paynejacob**
https://github.com/rancher/rancher/issues/34625

~Waiting on: https://github.com/rancher/image-mirror/pull/154~

Tested on k3s ha, works as expected

### CODE REVIEW

File: packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml [hunk 1]
```diff
@@ -10,13 +10,15 @@ spec:
       name: {{ .Release.Name }}-rke2-journald-aggregator
   template:
     metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/loggings/rke2/configmap.yaml") . | sha256sum }}
```
Comment (nickgerace) [inline packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml:14]: What does this do? It appears to hash the RKE2 ConfigMap contents and use it as a checksum?

**Review by nickgerace — APPROVED** [review]
Overall, LGTM. The backports appear to have been migrated without malformation.

**Comment by snasovich** [comment]
This should be good to merge based on @nickgerace 's comment and referenced image-mirror PR merged.

File: packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml (hunk 1 above)
Comment (paynejacob) [reply packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml:14]: It forces a redeploy of the aggregator pods whenever the configuration is updated.  Older versions of logging would not restart these pods and required the users to manually redeploy the daemonsets.

Comment (nickgerace) [reply packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml:14]: Smart... love it.

**Review by rmweir — APPROVED** [review]
I didn't see any of the original 2.6 PRs linked but if this is just a backport and has been tested, it should be fine. LGTM.

File: index.yaml [hunk 2]
```diff
@@ -2330,6 +2330,29 @@ entries:
...
+    digest: 27f8c4224711907728b9f5263874376890192db0e4dd11e28cc0a5ae463658e6
+    icon: https://charts.rancher.io/assets/logos/logging.svg
+    keywords:
+    - logging
+    - monitoring
+    - security
+    name: rancher-logging
+    urls:
+    - assets/rancher-logging/rancher-logging-3.9.400+up3.9.4.tgz
+    version: 3.9.400+up3.9.4
```
Comment (cbron) [inline index.yaml:2355]: this is just a re-shuffling

**Review by cbron — APPROVED** [review]
Didn't review code itself, just assets/index. If this is clean backport and all images exist we should be good. Just do a basic dev-validate before sending to qa.

**Pull request merged**